xla/service/gpu/ir_emitter_triton.cc

```diff
@@ -122,14 +126,21 @@ limitations under the License.
 #include "tsl/platform/path.h"
 #include "tsl/platform/status.h"
 #include "tsl/platform/statusor.h"
+#ifdef TENSORFLOW_USE_ROCM
```

Comment:
I think it'd be good if possible to minimize the number of ifdefs, so maybe ifdefs used everywhere at the top, then ```
#ifdef GOOGLE_CUDA
<includes> 
#ifdef TENSORFLOW_USE_ROCM
<includes>
``` or similar.